README.md

```diff
@@ -109,3 +130,80 @@ This field can be used to add your own definitions and rules to the Makefile.
 The text in this field is copied into the Makefile mostly verbatim, with one exception:
 Since YAML does not like tabs for indentation, we allow rule recipes to be indented with spaces.
 This indentation will be replaced with tabs before writing it into the actual Makefile.
+
+### `githubWorkflows`
+
+This is how a minimal and complete workflow configuration would look like:
+
+```yaml
+githubWorkflows:
+  global:
+    ignorePaths:
+      - "**.md" # all Markdown files
+  ci:
+    enabled: true
+    coveralls: true
+  license:
+    enabled: true
+    patterns: [ "**.go" ] # all Go files
```

Comment:
If I'm not mistaken, those patterns are consumed by shell scripts, where patterns work differently than on GitHub Actions file matches. On my local bash, `echo **.go` is the same as `echo *.go`. Double-star globs are only supported on zsh, and even then, listing all Go files would be expressed as `echo *.go **/*.go`.